conda/recipes/pynvjitlink/meta.yaml

```
@@ -0,0 +1,69 @@
+# Copyright (c) 2018-2023, NVIDIA CORPORATION.
+
+{% set py_version = environ['CONDA_PY'] %}
+{% set cuda_version = '.'.join(environ['RAPIDS_CUDA_VERSION'].split('.')[:2]) %}
+{% set cuda_major = cuda_version.split('.')[0] %}
+{% set date_string = environ['RAPIDS_DATE_STRING'] %}
+
+{% set data = load_file_data("pyproject.toml") %}
+
+package:
+  name: pynvjitlink
+  version: {{ data.get("project", {}).get("version", "") }}
+
+source:
+  path: ../../..
+
+build:
+  noarch: python
+  script:
+    - {{ PYTHON }} -m pip install . -vv --config-settings=cmake.args=-DCUDAToolkit_ROOT=/usr/local/cuda
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Would add the CUDA compiler here

```suggestion
    - {{ compiler('cxx') }}
    - {{ compiler('cuda') }}
```